test/card-test.js

```diff
@@ -59,12 +99,105 @@ describe('cards interactions', function () {
     let msg2 = card.createdBy + " == " + createdByBefore +
       "\n\t(createdBy should not change once Card is instantiated)";
     let msg3 = card.lastInteraction + " != " + lastInteractionBefore +
-      "\n\t(lastInteraction should update after Card#updateMetadata()" +
-      " method is evoked)";
+    "\n\t(lastInteraction should update after Card#updateMetadata()" +
+    " method is evoked)";
```

Comment:
For lines 102-103, it is unclear why these lines are being shortened up to the same vertical alignment as line 101, which represents the starting line for the run-on content that is on lines 102-103.  Especially in light of the allowance for the original style on lines 99-100 remaining the same. This change creates a discontinuity in styles.